### DISCUSSION TITLE
How to set running port in prod mode

### DISCUSSION THREAD

**Discussion by kaikaibenkai** [nuxt/nuxt#20839] [Questions] [answered]
I'm m using `./output/server/index.mjs`, how to change the default port `3000`?

thanks

**Comment by warflash** [comment, marked as answer]
https://nuxt.com/docs/getting-started/deployment#configuring-defaults-at-runtime so set
- NITRO_PORT or PORT (defaults to 3000)
- NITRO_HOST or HOST (defaults to '0.0.0.0')

**Reply by kaikaibenkai** [reply]
Can appending to `.env` file take effect? or must it be set in Bash

**Reply by maou-shonen** [reply]
Currently the .env file only takes effect in the development environment
